sled-agent/src/services.rs

```diff
@@ -4605,6 +4668,16 @@ mod test {
         let delete_vnic_ctx = MockDladm::delete_vnic_context();
         delete_vnic_ctx.expect().returning(|_| Ok(()));
 
+        // Also send a message to the metrics task that the VNIC has been
+        // deleted.
+        mgr.inner
+            .sled_info
+            .get()
+            .unwrap()
+            .metrics_queue
+            .untrack_vnic("oxControlService0")
```

Comment:
minor: even for testing, can this be derived from existing data? Just wondering aloud.

Reply:
We probably can, by reaching into the `mgr` in this function. If not, I could at least make this a const.

Reply:
Done in f9d4b5c3f